internal/utils/templates/init_config.toml

```diff
@@ -63,13 +72,29 @@ double_confirm_changes = true
 # If enabled, users need to confirm their email address before signing in.
 enable_confirmations = false
 
+[auth.sms]
+# Allow/disallow new user signups via SMS to your project.
+enable_signup = true
+# If enabled, users need to confirm their phone number before signing in.
+enable_confirmations = false
+
+# Configure one of the supported SMS providers: `twilio`, `messagebird`, `textlocal`, `vonage`.
+[auth.sms.twilio]
+enabled = false
+account_sid = ""
+message_service_sid = ""
+# DO NOT commit your Twilio auth token to git. You can leave this field blank and define the
+# following environment variable instead: SUPABASE_AUTH_SMS_TWILIO_AUTH_TOKEN
+auth_token = ""
```

Comment:
Why not use `env(SUPABASE_AUTH_SMS_TWILIO_AUTH_TOKEN)`?

Reply:
Sure, that sounds better because it's more explicit for the user.